index.js

```diff
@@ -20,10 +21,22 @@ async function run() {
     const title = pullRequest.title;
     const description = pullRequest.body;
     const branch = pullRequest.head.ref;
-    const prefixes = core.getInput("prefixes", {required: true});
+
+    const titleExceptionPrefixes = core.getInput("exceptionTitlePrefixes", { required: true });
+    core.info(`Found titleExceptionPrefixes: ${titleExceptionPrefixes}`);
+    for (const titleExceptionPrefix of titleExceptionPrefixes) {
```

Comment:
will `titleExceptionPrefixes` not be a comma separated list like `prefixes` below?

Reply:
Oh man good catch